ckanext/datarequests/tests/test_actions.py

```diff
@@ -36,6 +36,7 @@ def setUp(self):
         actions.USERS_CACHE = {}
         actions.tk.ObjectNotFound = self._tk.ObjectNotFound
         actions.tk.ValidationError = self._tk.ValidationError
+        actions.tk.h.closing_circumstances_enabled = False
```

Comment:
do we have a unit test where this is True?

Reply:
No unfortunately, I only had enough time to create BDD tests which were timeboxed to 4 hours https://salsadigital.atlassian.net/browse/DQL2-40.
We can create another story for unit tests as it probably will need to be done during the open-source giveback story https://salsadigital.atlassian.net/browse/DQL2-48